### ISSUE TITLE
resolveId() - absolute paths of external modules are not to be converted to relative paths

### ISSUE THREAD

**Issue by fanick1** [rollup/rollup#3940]
### Expected Behavior / Situation

for some arbitrary module id a plugin with the `resolveId()` hook resolves this id to:
```
{
  id: '/some/absolute/path.haschode123.min.js',
  external: true
}
```

then this path is tranformed into the output file as-is

`import foo from '/some/absolute/path.hashcode123.min.js'`

### Actual Behavior / Situation

this path is mangled in the output file into something like

`import foo from '../../../../some/absolute/path.hashcode123.min.js'`

which works in my case now only by accident (there are enough ../ to get to the root path in the browser)

### Modification Proposal

do not `path.relative()` the id returned from `resolveId` when the id is marked as external.

using `output.paths` is not an option for me - I don't know the paths up front and I need to do some tricky resolutions

**Comment by fanick1** [comment]
I was lucky enough to run into a case where the rollup does not put enough '../' into the path and now the module (let's say the module in the example `path.hashocde123.min.js`) cannot be found because I am serving one module too deep in the URL structure.
A quick workaround I had to do is to prepend the module id returned from the `resolveId()` with few '../' :
```
{
  id: '../../../../../../../../../some/absolute/path.haschode123.min.js',
  external: true
}
```
This of course still leaves room for error. I really think that external modules with a leading '/' in the name should be treated as absolute paths.

**Comment by fanick1** [comment]
I believe this is the same issue as in #3831.

**Comment by lukastaegert** [comment]
Fix at fix at #4021
